glide-core/src/client/value_conversion.rs

```diff
@@ -673,6 +674,237 @@ pub(crate) fn convert_to_expected_type(
             )
                 .into()),
         },
+        // `XINFO STREAM` returns nested maps with different types of data
+        /* RESP2 response example
+
+        1) "length"
+        2) (integer) 2
+        ...
+        13) "recorded-first-entry-id"
+        14) "1719710679916-0"
+        15) "entries"
+        16) 1) 1) "1719710679916-0"
+               2) 1) "foo"
+                  2) "bar"
+                  3) "foo"
+                  4) "bar2"
+                  5) "some"
+                  6) "value"
+            2) 1) "1719710688676-0"
+               2) 1) "foo"
+                  2) "bar2"
+        17) "groups"
+        18) 1)  1) "name"
+                2) "mygroup"
+                ...
+                9) "pel-count"
+               10) (integer) 2
+               11) "pending"
+               12) 1) 1) "1719710679916-0"
+                      2) "Alice"
+                      3) (integer) 1719710707260
+                      4) (integer) 1
+                   2) 1) "1719710688676-0"
+                      2) "Alice"
+                      3) (integer) 1719710718373
+                      4) (integer) 1
+               13) "consumers"
+               14) 1) 1) "name"
+                      2) "Alice"
+                      ...
+                      7) "pel-count"
+                      8) (integer) 2
+                      9) "pending"
+                      10) 1) 1) "1719710679916-0"
+                             2) (integer) 1719710707260
+                             3) (integer) 1
+                          2) 1) "1719710688676-0"
+                             2) (integer) 1719710718373
+                             3) (integer) 1
+        
+        RESP3 response example
+
+        1# "length" => (integer) 2
```

Comment:
let's remove the RESP3 example here... we aren't doing any conversion so it's kind of pointless